akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -209,25 +209,7 @@ See @ref:[Cluster Node Roles](typed/cluster.md#node-roles)
 <a id="min-members"></a>
 ## How To Startup when Cluster Size Reached
 
-A common use case is to start actors after the cluster has been initialized,
-members have joined, and the cluster has reached a certain size.
-
-With a configuration option you can define required number of members
-before the leader changes member status of 'Joining' members to 'Up'.:
-
-```
-akka.cluster.min-nr-of-members = 3
-```
-
-In a similar way you can define required number of members of a certain role
-before the leader changes member status of 'Joining' members to 'Up'.:
-
-```
-akka.cluster.role {
-  frontend.min-nr-of-members = 1
-  backend.min-nr-of-members = 2
-}
-```
+See @ref:[fu](typed/cluster.md#how-to-startup-when-a-cluster-size-is-reached)
 
 ## How To Startup when Member is Up
 
```

Comment:
Missing crossrefs to typed from up top in this file

Reply:
I was planning to put that in a separate commit for a few of them, and keep this as much just sharding as possible. Will do that PR next.

Reply:
I'm not really sure what you mean here?

Reply:
Hmmm, maybe the comment ended up on the wrong file. Not sure.